include/mbedtls/nist_kw.h

```diff
@@ -0,0 +1,174 @@
+/**
+ * \file nist_kw.h
+ *
+ * \brief This file provides API for key wrapping(KW)
+ *        and key wrapping with padding (KWP) as defined in NIST SP 800-38F
+ *        https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-38F.pdf
+ *
+ * Key-wrapping specifies a deterministic authenticated-encryption mode of operation,
+ * according to <em>NIST SP 800-38F: Recommendation for Block Cipher Modes of Operation:
+ * Methods for Key Wrapping</em>.
+ * Its purpose is to protect cryptographic keys.
+ *
+ * It's equivalent is RFC 3394 for KW, and RFC 5649 for KWP.
+ * https://tools.ietf.org/html/rfc3394
+ * https://tools.ietf.org/html/rfc5649
+ *
+ * Note: RFC 5649 defines the object IDs for KW and KWP, which is not supported.
```

Comment:
Why not add the definitions to `oid.h`?

Reply:
Because it's out of scope of this PR.
We can add it to oid.h, but we aren't implementing anything that uses ASN.1 notation for key wrapping

Reply:
We don't, but library users may. I can't think of a good reason not to put the definitions in `oid.h`.

Reply:
If we don't have any way to test\verify it, I prefer not having it

Reply:
They're just constants. You can test them exhaustively by comparing the value of the constant with another copy of the same constant.